docs/source/databases/orientdb.rst

```diff
@@ -1,2 +1,21 @@
 OrientDB
 ========
+
+.. TODO: Give more insight into how schema generation works for OrientDB, (i.e. how do vertex, and
+         edge classes get mapped to OrientDB constructs).
+
+The best way to integrate the compiler with OrientDB is by compiling to MATCH, our name for the
+SQL dialect that OrientDB uses. Since the compiler was originally built to target an OrientDB
```

Comment:
Just remove the `Since the compiler was originally built` part. It makes it look like if you don't care about OrientDB this library is not for you.